apps/catalog/catalog/src/app/marketplace/title/avails/avails.component.ts

```diff
@@ -133,7 +140,7 @@ export class MarketplaceMovieAvailsComponent implements AfterViewInit, OnDestroy
 
   private async init() {
 
-    const contracts = await this.contractService.getValue(ref => ref.where('titleId', '==', this.movie.id).where('status', '==', 'accepted'));
+    const contracts = await this.contractService.getValue(ref => ref.where('titleId', '==', this.route.snapshot.params.movieId).where('status', '==', 'accepted'));
```

Comment:
create a local variable for readability : 
```suggestion
const movieId = this.route.snapshot.params.movieId;
const contracts = await this.contractService.getValue(ref => ref.where('titleId', '==', movieId ).where('status', '==', 'accepted'));
```